libs/gl-client-py/glclient/lsps.py

```diff
@@ -91,29 +91,32 @@ class NoParams:
 
 
 class LspClient:
-    def __init__(self, native: native.LspClient, peer_id: t.Union[bytes, str]):
+    def __init__(self, native: native.LspClient):
         self._native = native
-        self._peer_id: bytes = parse_and_validate_peer_id(peer_id)
 
     def _rpc_call(
         self,
-        peer_id: bytes,
+        peer_id: str,
         method_name: str,
         param_json: bytes,
         json_rpc_id: t.Optional[str] = None,
     ) -> bytes:
         logger.debug("Request lsp to peer %s and method %s", peer_id, method_name)
+        peer_id_bytes = bytes.fromhex(peer_id)
         if json_rpc_id is None:
-            return self._native.rpc_call(peer_id, method_name, param_json)
+            return self._native.rpc_call(peer_id_bytes, method_name, param_json)
         else:
             return self._native.rpc_call_with_json_rpc_id(
-                peer_id, method_name, param_json, json_rpc_id=json_rpc_id
+                peer_id_bytes, method_name, param_json, json_rpc_id=json_rpc_id
             )
 
-    def list_protocols(self, json_rpc_id: t.Optional[str] = None) -> ProtocolList:
+    def list_lsp_servers(self) -> t.List[bytes]:
```

Comment:
I think this return value should be `t.List[str]` now

Reply:
Thanks for noticing.

FIxed